src/Microsoft.Health.Dicom.Core/Features/Diagnostic/LogForwarderExtensions.cs

```diff
@@ -44,4 +46,28 @@ internal static class LogForwarderExtensions
 
         telemetryClient.TrackTrace(telemetry);
     }
+
+    /// <summary>
+    /// Emits a trace log with forwarding flag set and adds properties from instanceIdentifier as properties to telemetry.
+    /// </summary>
+    /// <param name="telemetryClient">client to use to emit the trace</param>
+    /// <param name="message">message to set on the trace log</param>
+    /// <param name="operationId">operation id</param>
+    /// <param name="input">Input payload to pass to the forward logger</param>
+    public static void ForwardLogTrace(
+        this TelemetryClient telemetryClient,
+        string message,
+        string operationId,
+        string input)
+    {
+        EnsureArg.IsNotNull(telemetryClient, nameof(telemetryClient));
+        EnsureArg.IsNotNull(message, nameof(message));
+
+        var telemetry = new TraceTelemetry(message);
+        telemetry.Properties.Add(InputPayload, input);
```

Comment:
Can this really be so generic to include input? The operation input contains data the function uses for checkpointing and data the user doesn't know about. Do we really want to expose this via the logs?

Do we include the input for the web apis?

Reply:
What is the requirement? We need to record what is being updated? If so:
- Record the user's input as part of the audit log AND/OR
- Record an "input" payload to telemetry that is tailored to someone reading it via the logs (i.e. it isn't coupled with the function input)